### ISSUE TITLE
Submit plugin to the TeamCity plugin registry

### ISSUE THREAD

**Issue by dploeger** [jonnyzzz/TeamCity.Node#183]
It would be nice, if this plugin could be available in Jetbrains' [plugin registry](https://plugins.jetbrains.com/)

**Comment by TheBoneJarmer** [comment]
I agree entirely. To my biggest surprise TeamCity does not ship with a nodejs build runner, which I find hardly acceptable. I mean, with JavaScript and NodeJS being so popular nowadays you'd expect at least this to be available..
